http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -689,15 +689,15 @@ object Config {
     )
   }
 
-  implicit private val rstudioAppConfigReader: ValueReader[RStudioAppConfig] = ValueReader.relative { config =>
-    RStudioAppConfig(
+  implicit private val aouAppConfigReader: ValueReader[AllowedAppConfig] = ValueReader.relative { config =>
+    AllowedAppConfig(
       config.as[ChartName]("chartName"),
-      config.as[ChartVersion]("chartVersion"),
+      config.as[ChartVersion]("rstudioChartVersion"),
+      config.as[ChartVersion]("sasChartVersion"),
       config.as[NamespaceNameSuffix]("namespaceNameSuffix"),
       config.as[ReleaseNameSuffix]("releaseNameSuffix"),
       config.as[List[ServiceConfig]]("services"),
       config.as[ServiceAccountName]("serviceAccountName"),
-      config.as[Boolean]("enabled"),
```

Comment:
This make me think that there is likely a terra helm file PR that should be done with this, see https://github.com/broadinstitute/terra-helmfile/blob/34f228ec6e4f39ee4195ddb784015ee98fcdebeb/values/app/leonardo/live/prod.yaml#L167

Reply:
for sure! I'll remove that once the PR is in!